contracts/SnappAuctionChallenge.sol

```diff
@@ -143,7 +137,7 @@ contract SnappAuctionChallenge {
             (uint leaf, bytes memory proof) = getBuyBalanceAndProof(merklePaths, i);
             Merkle.checkMembership(
                 bytes32(leaf), order.account * NUM_TOKENS + order.buyToken, stateRoot, proof, ACCOUNT_HEIGHT + TOKEN_HEIGHT
-            );
+            );  // This would just be a boolean statement which isn't assigned to anything?
```

Comment:
I had to remove the assert, since otherwise my tests wouldn't have been passing, but you are right, they should be wrapped in a require